### PR TITLE
Combine TDK demos under one roof

### PR DESCRIPTION
**Pull request by mgramin**

### CODE REVIEW

File: postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
+# TDK PostgreSQL Demo
+
+
+[Pagila](https://github.com/devrimgunduz/pagila) is a sample database schema that can be used in MySQL tutorials, articles, demos, etc.
```
Comment (kurumar) [inline postgres/README.md:4]: Is MySQL mentioned here by mistake?

File: postgres/README.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+```shell
+git clone https://github.com/synthesized-io/pagila-tdk-demo
+cd pagila-tdk-demo/postgres
+```
+
+Prepare an [inventory file](https://docs.synthesized.io/tdk/latest/user_guide/reference/inventory) - [inventory.yaml](inventory.yaml).
+
+
+## Generation of the empty database from scratch
```
Comment (kurumar) [inline postgres/README.md:14]: I suggest renaming the title to:

## Generation based on the empty schema

File: mysql/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# TDK MySQL Demo
+
+[Sakila](https://dev.mysql.com/doc/sakila/en/) is a sample database schema that can be used in MySQL tutorials, articles, demos, etc.
+
+```shell
+git clone https://github.com/synthesized-io/pagila-tdk-demo
+cd pagila-tdk-demo/mysql
+```
+
+Prepare an [inventory file](https://docs.synthesized.io/tdk/latest/user_guide/reference/inventory) - [inventory.yaml](inventory.yaml).
```
Comment (kurumar) [inline mysql/README.md:10]: What does the phrase "Prepare an inventory file" mean here? Should the user do something? I suggest removing it or to make it more clear.

**Review by kurumar — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,71 +1,27 @@
...
 
-A simple (but no less powerful) demo for data generation (masking and subsetting as well) for the legendary Pagila database schema using Synthesized TDK.
+A simple (but no less powerful) demo for data generation, masking and subsetting for different databases and options using [Synthesized TDK](https://docs.synthesized.io/tdk/latest/?utm_source=github&utm_medium=devrel&utm_campaign=datagen).
 
-[Pagila](https://github.com/devrimgunduz/pagila) is a sample database schema that can be used in tutorials, articles, demos, etc.
 
-[Synthesized TDK](https://docs.synthesized.io/tdk/latest/?utm_source=habr&utm_medium=devrel&utm_campaign=datagen) provides database generation and masking capabilities.
+## Preamble
 
+https://github.com/synthesized-io/pagila-data-generation
```
[the remark this answers is not in the document]
Comment (mgramin) [reply README.md]: Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)

File: README.md [hunk 5]
```diff
@@ -1,71 +1,27 @@
...
-```
-
-### Subsetting of the existing data
-
-```
-export CONFIG_FILE=config_subsetting.tdk.yaml
-docker-compose -f docker-compose.yaml -f input-db.yaml run tdk
-```
 
 ## Troubleshooting and advises
```
[the remark this answers is not in the document]
Comment (mgramin) [reply README.md:38]: Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)

File: README.md [hunk 6]
```diff
@@ -1,71 +1,27 @@
-# pagila-tdk-demo
+# TDK Demo
 
-A simple (but no less powerful) demo for data generation (masking and subsetting as well) for the legendary Pagila database schema using Synthesized TDK.
+A simple (but no less powerful) demo for data generation, masking and subsetting for different databases and options using [Synthesized TDK](https://docs.synthesized.io/tdk/latest/?utm_source=github&utm_medium=devrel&utm_campaign=datagen).
```
[the remark this answers is not in the document]
Comment (mgramin) [reply README.md]: Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)

File: README.md [hunk 7]
```diff
@@ -1,71 +1,27 @@
...
-[Synthesized TDK](https://docs.synthesized.io/tdk/latest/?utm_source=habr&utm_medium=devrel&utm_campaign=datagen) provides database generation and masking capabilities.
+## Preamble
 
+https://github.com/synthesized-io/pagila-data-generation
+https://www.synthesized.io/post/test-data-generation-there-and-back
 
-## How to start
 
-### Generation of the empty database from scratch
+## Demos
```
[the remark this answers is not in the document]
Comment (mgramin) [reply README.md:6]: Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)

File: README.md [hunk 8]
```diff
@@ -1,71 +1,27 @@
...
-```
+## Tools
 
-```
- count
---------
- 20000
-(1 row)
-```
+- [Synthesized TDK](https://docs.synthesized.io/tdk/latest/?utm_source=github&utm_medium=devrel&utm_campaign=datagen) provides database generation and masking capabilities.
```
[the remark this answers is not in the document]
Comment (mgramin) [reply README.md:15]: Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)

File: postgres/README.md (hunk 2 above)
Comment (mgramin) [reply postgres/README.md]: Fixed in [30b6f4e](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/30b6f4eb179902e9925d50ec80550b431a1012ca)

File: mysql/README.md (hunk 3 above)
Comment (mgramin) [reply mysql/README.md]: Fixed in [384d5ba](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/384d5ba92ac472b127fd1688589afe1251a3425f)

File: postgres/README.md (hunk 1 above)
Comment (mgramin) [reply postgres/README.md]: Fixed in [384d5ba](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/384d5ba92ac472b127fd1688589afe1251a3425f)

File: postgres/README.md [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+
+
+[Pagila](https://github.com/devrimgunduz/pagila) is a sample database schema that can be used in MySQL tutorials, articles, demos, etc.
+
+```shell
+git clone https://github.com/synthesized-io/pagila-tdk-demo
+cd pagila-tdk-demo/postgres
+```
+
+Prepare an [inventory file](https://docs.synthesized.io/tdk/latest/user_guide/reference/inventory) - [inventory.yaml](inventory.yaml).
```
[the remark this answers is not in the document]
Comment (mgramin) [reply postgres/README.md]: Fixed in [384d5ba](https://github.com/synthesized-io/pagila-tdk-demo/pull/12/commits/384d5ba92ac472b127fd1688589afe1251a3425f)

**Review by kurumar — APPROVED** [review]

**Pull request merged**